ui/packages/consul-ui/app/styles/debug.scss

```diff
@@ -4,7 +4,14 @@
 // temporary component debugging setup
 @import 'consul-ui/components/main-nav-vertical/debug';
 
+.is-debug .brand-loader {
+  display: none !important;
+}
```

Comment:
@johncowen Given that the eng docs only show locally, I think this solution is ok because this style is only loaded when we're a) local, b) looking at the docs and c) the docs css has been loaded (so we can hide the loading state).

Have I missed anything with this approach?

Reply:
Don't think so, thanks for extra explanation!

Ping me when this is out of draft and I can give it a 👍